remake_wqp.yml

```diff
@@ -48,3 +50,10 @@ targets:
     
   secchi.clean:
     command: munge_wqp(secchi.merged)
+    depends: 
+      - scripts/download_munge_wqp.R
```

Comment:
wasn't sure - I added that and it re-ran after a change vs not before...will figure it out.